lib/api/src/grpc/conversions.rs

```diff
@@ -332,10 +332,12 @@ impl From<segment::types::WithPayloadInterface> for WithPayloadSelector {
 impl From<QuantizationSearchParams> for segment::types::QuantizationSearchParams {
     fn from(params: QuantizationSearchParams) -> Self {
         Self {
-            ignore: params.ignore.unwrap_or(default_quantization_ignore_value()),
+            ignore: params
+                .ignore
+                .unwrap_or_else(default_quantization_ignore_value),
             rescore: params
                 .rescore
-                .unwrap_or(default_quantization_rescore_value()),
+                .unwrap_or_else(default_quantization_rescore_value),
```

Comment:
I think leaving this as is and changing the invoked function to `const fn` or to a `const` value is better. It would be evaluated at compile time, removing any function calls.

I expect that the Rust compiler already eliminates this function call in release builds, which would mean that your change has no effect in the release binary, though I haven't verified this. But then I still think the approach I described above is better.

The same goes for all the other places you changed this.

Reply:
> I think leaving this as is and changing the invoked function to `const fn`

Will do.